department-of-veterans-affairs/va.gov-cms
# VAMCS-7746 more claro mob session fixes

## Description

Relates to #7746. Builds on the work in PRs #7866 and #7831 in order to ensure correct vars are used and no further regressions introduced.

- Updates dependencies for vagovclaro and vagovadmin as directed by dependabot alerts.
- Node link reports have an icon indicating its a dropdown again
- Updates link color in alerts for a11y purposes
- Hides redundant 'status message' title
- Fixes KB page regression... paragraph proofing style no longer applied.
- Adds media query for content layout on very large screens (>1920px) 
- Adds notes about xdebug in ddev to DEBUGGING.md

## Testing done
vagovclaro
```
yarn audit
yarn audit v1.22.17
0 vulnerabilities found - Packages audited: 687
:sparkles:  Done in 0.52s.
```
vagovadmin
```
yarn audit
yarn audit v1.22.17
0 vulnerabilities found - Packages audited: 687
✨  Done in 0.54s.
```


## Screenshots
<img width="1862" alt="Screen Shot 2022-02-03 at 3 56 29 PM" src="https://user-images.githubusercontent.com/11279744/152449093-266c039d-84d6-4a22-b472-bb302d4c7815.png">
<img width="1869" alt="Screen Shot 2022-02-03 at 3 56 35 PM" src="https://user-images.githubusercontent.com/11279744/152449094-9d36948b-95d2-480f-aa1f-95fbf3af791a.png">
<img width="999" alt="Screen Shot 2022-02-03 at 3 56 52 PM" src="https://user-images.githubusercontent.com/11279744/152449096-5957046d-4f16-45e4-a520-0147cb7ce0f3.png">


## QA steps
`/node/3776/edit`
`/help`

login, edit node 3776 (tacoma vet center), see that node link report has a caret again, link color in alerts pass a11y. See that the status message indicating content lock no longer has a redundant "Status message" title.

If you have a very wide monitor, note that the main content column on edit forms gets slightly wider above 1920px. (You can resize overall window or resize inspector pane to trigger breakpoints)

click the Knowledge Base link in the toolbar, see that the content on the /help page looks as it is supposed to. No visible boxes around paragraphs.


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
